stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/OnrampCheckoutResult.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.crypto.onramp.model
+
+/**
+ * Result of a checkout operation.
+ */
+sealed interface OnrampCheckoutResult {
+    /**
+     * Checkout completed successfully.
+     */
+    data object Completed : OnrampCheckoutResult
```

Comment:
```suggestion
    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
    class Completed internal constructor() : OnrampCheckoutResult
```
etc. for each of these